src/keycloak/chart/templates/istio-admin.yaml

```diff
@@ -37,22 +38,33 @@ spec:
             - monitoring
     - to:
         - operation:
-            ports: 
+            ports:
               - "8080"
             paths:
               # Never allow anonymous client registration except from the pepr-system namespace
               # This is another fallback protection, as the KC policy already blocks it
               - "/realms/{{ .Values.realm }}/clients-registrations/*"
       from:
         - source:
-            notNamespaces: 
+            notNamespaces:
               - "pepr-system"
+    - to:
+        - operation:
+            ports:
+              - "8080"
+            paths:
+              - "/admin/realms/{{ .Values.realm }}/clients"
```

Comment:
Update the design: mention that we can't block clients because Keycloak doesn't have a method to get a Client by the Client ID. This happens through the collection endpoint.